stable/rabbitmq/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: rabbitmq
-version: 6.2.6
```

Comment:
Can you bump the chart version in a minor? It is a new feature

Reply:
Yes, is true, I bump a new version